solutions/usaco-893.mdx

```diff
@@ -0,0 +1,123 @@
+---
+id: usaco-893
+source: USACO Bronze 2019 January
+title: Guess the Animal
+author: Brad Ma
+---
+
+[Official Analysis (C++)](https://codeforces.com/blog/entry/22712)
+
+<LanguageSection>
+  
+<CPPSection>
+  
+```cpp
```

Comment:
would be nice to add comments to this code instead of just copy pasting

same for java sol